.golangci.yml

```diff
@@ -1,3 +1,6 @@
+# Options for analysis running.
+run:
+  timeout: 2m
```

Comment:
https://golangci-lint.run/usage/configuration/

```yml
# Options for analysis running.
run:
  # The default concurrency value is the number of available CPU.
  concurrency: 4
  # Timeout for analysis, e.g. 30s, 5m.
  # Default: 1m
  timeout: 5m
```

I'll double it for now and see what happens.